no_mutations.md

```diff
@@ -11,20 +11,266 @@ kernelspec:
+# Do you really need mutations?
+
+In tree sequences, the genetic genealogy exists independently of the mutations that
+generate genetic variation. This tutorial aims to illustrate
+cases where the genealogy on its own may be sufficent; a number of
+analyses do not actually require genetic variation.
+
+This applies particularly to cases where you generate tree sequences by simulation.
+In this case, although simulating variable sites allows you to generate actual genetic
+data, it's worth considering the following points:
+
+1. Neutral mutations and sites can be added to a genealogy later, if you find they *are*
+needed
+2. Simulating sites and mutations increases memory requirements and takes marginally
+more CPU time
+3. Storing mutations increases the tree sequence file size
+4. Some genetic analyses can be performed equally well (or better!) on tree sequences
+that have no sites or mutations.
+
+To illustrate the first three points, we can use the [msprime](https://tskit.dev/msprime)
+{func}`~msprime.sim_mutations` function to add neutral sites and mutations onto an
+simulated mutationless tree sequence of 20 diploid individuals:
+
+```{code-cell} ipython3
+import msprime
+L = 1_000_000  # simulate 1 megabase length (could increase for a larger example)
+rho = mu = 1e-8  # Human-like recombination and mutation parameters
+n_subpops = 2
+subpop_size = 1e4
+migration_rate = 1e-4
+# Create a mutationless diploid tree sequence of n_subpops demes
+ts_no_mut = msprime.sim_ancestry(
+    samples={f"pop_{i}": 10 for i in range(n_subpops)},  # 10 samples from each subpop
+    demography=msprime.Demography.island_model([subpop_size] * n_subpops, migration_rate),
+    ploidy=2,
+    recombination_rate=rho,
+    sequence_length=L,
+    random_seed=123,
+)
+
+# Optionally, add neutral mutations later, after simulating. This takes some CPU time
+# (although it is usually fast compared to simulating the original tree sequence)
+ts_mutated = msprime.sim_mutations(ts_no_mut, rate=mu, random_seed=456)
+print(
+    "Adding mutations has increased the tree sequence file size by "
+    f"{ts_mutated.nbytes / ts_no_mut.nbytes * 100:.0f}%",
+)
+```
+
+:::{note}
+Above we have overlaid sites and mutations onto an existing tree sequence by simulation.
+It is also possible to overlay mutations in such a way as to generate a known pattern of
+genetic variation, using {meth}`tskit.Tree.map_mutations`.
+:::
+
+
+Although including mutations has increased the file size a fair bit in this example,
+unless you are running simulations with vast numbers of mutations, tree sequences are
+usually compact enough that the cost of including them shouldn't be prohibitive.
+Nevertheless, it may not be obvious that it is perfectly possible, indeed sometimes
+preferable, to perform genetic analyses on tree sequences that do not contain mutations
+and variable sites. This is the focus of the remainder of this tutorial.
```

Comment:
At first I was averse to saying "genetic analyses" here - it felt weird - but maybe that's the point, and I think it's ok?